crates/red_knot_python_semantic/resources/mdtest/descriptor_protocol.md

```diff
@@ -468,8 +529,7 @@ c.name = "new"
 # error: [invalid-assignment]
 c.name = None
 
-# TODO: this should be an error, but with a proper error message
-# error: [invalid-assignment] "Object of type `Literal[42]` is not assignable to attribute `name` of type `<bound method `name` of `C`>`"
+# error: [invalid-assignment] "Implicit shadowing of function `name`; annotate to make it explicit if this is intentional"
```

Comment:
This still deserves a TODO like before -- the error has changed but its still wrong, pending property support.